src/briefcase/platforms/linux/system.py

```diff
@@ -1038,21 +1038,25 @@ def _package_pkg(self, app: AppConfig, **kwargs):
 with details about the release.
 """
             )
-
+            
         # Generate the pkgbuild layout
-        pkgbuild_path = self.bundle_path(app) / "pkgbuild"
+        pkgbuild_path = self.build_path(app) / "pkgbuild"
```

Comment:
This isn't correct - it definitely needs to be `bundle_path`, or you'll generate code in a directory outside the Arch bundle.

Reply:
yes, we need to generate the code outside the `bundle_path` as the whole bundle_path will be archived for source tarball.

Reply:
This setup is going to lead to a briefcase file layout that looks like:

* build
   * myapp
     * arch
       * pkgbuild
         * ...
       * rolling
         * bootstrap
           * ... 
         * myapp-1.2.3
           * usr
             * ...

That's the wrong location for the `pkgbuild` folder. It should be inside the `rolling` folder. This is, at the very least, needed to support Docker builds, because Docker mounts `build/myapp/arch/rolling` as `/app`; so the `pkgbuild` folder won't be visible inside the Docker container. More generally, all the artefacts of the `arch/rolling` build should be in the `arch/rolling` folder.

Reply:
Thanks! I was confused about the directory structure. I will modify it.